packages/runtime/container-runtime/src/opLifecycle/outbox.ts

```diff
@@ -259,13 +259,40 @@ export class Outbox {
 			true /* disableGroupedBatching */,
 			resubmittingBatchId,
 		);
-		this.flushInternal(
+		this.flushInternalMainBatch(
```

Comment:
I thought we said BlobAttach batch could end up empty too?

Reply:
BlobAttach ops are resubmitted on a 1:1 basis so it's not needed as long as that assumption stays in place. I'll add a comment to the code.